examples/run/sycl/seq_example_sycl.sycl

```diff
@@ -129,143 +119,118 @@ int seq_run(const traccc::full_tracking_input_config& i_cfg,
+                    Clusterization (cpu)
+                -----------------------------*/
+
+                {
+                    traccc::performance::timer clusterization_timer_cpu(
+                        "Clusterization  (cpu)", elapsedTimes);
+                    measurements_per_event = ca(cells_per_event);
+                }
+
+                /*---------------------------------
+                    Spacepoint formation (cpu)
+                ---------------------------------*/
+
+                {
+                    traccc::performance::timer sf_timer_cpu(
+                        "Spacepoint formation  (cpu)", elapsedTimes);
+                    spacepoints_per_event = sf(measurements_per_event);
+                }
+            }
 
-        /*----------------------------
-          Track params estimation
-          ----------------------------*/
+            /*----------------------------
+                Seeding algorithm
+            ----------------------------*/
 
-        // SYCL
+            // SYCL
+            {
+                traccc::performance::timer seeding_timer("Seeding (sycl)",
+                                                         elapsedTimes);
+                seeds_sycl_buffer = sa_sycl(spacepoints_sycl_buffer);
+            }  // stop measuring seeding sycl timer
 
-        /*time*/ auto start_tp_estimating_sycl =
-            std::chrono::system_clock::now();
+            // CPU
 
-        auto params_sycl = tp_sycl(spacepoints_sycl_buffer, seeds_sycl_buffer);
+            if (run_cpu) {
+                traccc::performance::timer seeding_timer_cpu("Seeding  (cpu)",
+                                                             elapsedTimes);
+                seeds = sa(spacepoints_per_event);
+            }  // stop measuring seeding cpu timer
 
-        /*time*/ auto end_tp_estimating_sycl = std::chrono::system_clock::now();
-        /*time*/ std::chrono::duration<double> time_tp_estimating_sycl =
-            end_tp_estimating_sycl - start_tp_estimating_sycl;
-        /*time*/ tp_estimating_sycl += time_tp_estimating_sycl.count();
+            /*----------------------------
+            Track params estimation
+            ----------------------------*/
 
-        // CPU
+            // SYCL
+            {
+                traccc::performance::timer tp_timer("Track params (sycl)",
+                                                    elapsedTimes);
+                params_sycl =
+                    tp_sycl(spacepoints_sycl_buffer, seeds_sycl_buffer);
+            }  // stop measuring track params timer
 
-        traccc::track_params_estimation::output_type params;
-        if (run_cpu) {
-            /*time*/ auto start_tp_estimating_cpu =
-                std::chrono::system_clock::now();
+            // CPU
 
-            params = tp(spacepoints_per_event, seeds);
+            if (run_cpu) {
+                traccc::performance::timer tp_timer_cpu("Track params  (cpu)",
```

Comment:
Giving elaborate names for the variables is not bad, but you might as well just call them `t`. Since we don't need to pass them anywhere where their names would become useful.